xwininfo.rb

```diff
@@ -7,10 +7,9 @@ class Xwininfo < Formula
   sha256 "218eb0ea95bd8de7903dfaa26423820c523ad1598be0751d2d8b6a2c23b23ff8"
   # tag "linuxbrew"
 
-  depends_on "pkg-config" =>  :build
+  depends_on "pkg-config" => :build
   depends_on "libxcb"
   depends_on "xcb-util-wm"
-  # depends_on "libiconv" # currently does not compile under Linux
```

Comment:
We don't actually need `libiconv` on Linux IIRC, because the same functionality is handled by Glibc.